### PR TITLE
[ST07.1.6 - API] - Criar arquivo commons

### PR DESCRIPTION
**Pull request by RafaelFelice**
**1. Task Description:**

Create file commons

**2. Testing Procedures:**

it's not possible

**3. Comments:**

Create file commons

**4. Checklist review:**

- [x] The branch name follows the feature-title pattern
- [x] PR name follows the pattern [TASK] title-of-feature
- [x] The PR is signed by someone
- [x] PR has the correct label
- [x] Is the build green?
- [x] Can the branch be deleted?

### CODE REVIEW

File: serverest/tests/commons/Commons.robot [hunk 1]
```diff
@@ -0,0 +1,23 @@
+***Keywords***
+Create Users
+    [Documentation]     Used to create users.
+    [Arguments]         ${nome}     ${email}     ${password}     ${administrador}
+
+    &{USER}                             Create Dictionary
+    ...                                 nome= ${nome}
+    ...                                 email=${email} 
+    ...                                 password=${password}3
```
Comment (uLucasFraga) [inline serverest/tests/commons/Commons.robot]: What's that number "3"?

File: serverest/tests/commons/Commons.robot [hunk 2]
```diff
@@ -0,0 +1,23 @@
+***Keywords***
+Create Users
```
Comment (uLucasFraga) [inline serverest/tests/commons/Commons.robot]: @RafaelFelice this KW doesn't "create" anything.
I didn't understand the usage.

Could you explain to me, please? thanks.

File: serverest/tests/commons/Commons.robot [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    ...                                 password=${password}3
+    ...                                 administrador=${administrador}
+
+    Log                                 ${USER}
+    Set Test Variable                   ${USER}
+
+
+Get Token
+    [Documentation]     Used to get a token.
+    Create Users                        Phao  phao@qa.com  123  True
```
Comment (uLucasFraga) [inline serverest/tests/commons/Commons.robot]: I don't understand what you tried to do here, sorry.

File: serverest/tests/commons/Commons.robot [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    Login                               phao@qa.com  123
```
Comment (uLucasFraga) [inline serverest/tests/commons/Commons.robot]: @RafaelFelice I said we can't set a parameter on this KW, remember?

For example: phao

**Review by uLucasFraga — CHANGES_REQUESTED** [review]
@RafaelFelice you need to make the necessary fixes.
This PR doesn't make sense for me.

**Review by uLucasFraga — APPROVED** [review]

**Review by LucasFelice — APPROVED** [review]

**Pull request merged**